src/commands/seek.js

```diff
@@ -0,0 +1,113 @@
+const { Command } = require(`discord-akairo`);
+const { MessageEmbed } = require(`discord.js`);
+const PlayCommand = require(`./play`);
+
+/*eslint class-methods-use-this: ["error", { "exceptMethods": ["exec", "execSlash"] }] */
+class SeekCommand extends Command {
+    constructor() {
+        super(`seek`, {
+            aliases: [`seek`],
+            args: [
+                {
+                    id: `time`,
+                    type: `string`,
+                }
+            ],
+            slash: true,
+            slashOptions: [
+                {
+                    name: 'time',
+                    type: 'STRING',
+                    description: 'The timestamp (xx:xx) of where you want to seek',
+                    required: true
+                },
+            ],
+            category: `music`,
+            description: `Seeks to a specified time in the song`,
+            channel: `guild`
+        });
+    }
+
+    exec() {
+        return;
+    }
+
+    execSlash(message, args) {
+        if (!args.time.includes(`:`) || args.time.length <= 2) {
+            return message.interaction.reply({
+                embeds: [
+                    new MessageEmbed()
+                        .setDescription(`<:cross:729019052571492434> Please format the timestamp as xx:xx (like a YouTube video)\nEx: \`4:05\`, \`10:30\`, \`:05\``)
+                        .setColor(`#FF3838`)
+                ],
+                ephemeral: true
+            });
+        } else if (isNaN(args.time.substring(0, args.time.indexOf(`:`))) || isNaN(args.time.substring(args.time.indexOf(`:`) + 1))) {
+            return message.interaction.reply({
+                embeds: [
+                    new MessageEmbed()
+                        .setDescription(`<:cross:729019052571492434> There seems to be non-number characters in your input`)
+                        .setColor(`#FF3838`)
+                ],
+                ephemeral: true
+            });
+        }
+
+        let totalSeconds = 0;
+
+        // Convert minutes
+        totalSeconds += parseInt(args.time.substring(0, args.time.indexOf(`:`))) * 60 || 0;
```

Comment:
> __[ESLint]__ Missing radix parameter. ([view](https://sider.review/gh/repos/219657356/pulls/106#issue-6493167))
>
> <details>
>
> | Rule | Severity | Recommended |
> | ---- | -------- | ----------- |
> | `radix` | `warn` | No |
>
> References:
> - <https://eslint.org/docs/rules/radix>
>
> You can [close this issue](https://sider.review/gh/repos/219657356/pulls/106#issue-6493167) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Not necessary because I'm using newer than ES5